solara/website/pages/documentation/examples/utilities/todo.py

```diff
@@ -183,14 +136,16 @@ def TodoStatus():
         solara.Success("All done, awesome!", dense=True)
 
 
+state = State(initial_items)
+
+
 @solara.component
 def Page():
     with solara.Card("Todo list", style="min-width: 500px"):
```

Comment:
Please consider doing a separate Todo component card and show how it can be reused. Again, this may be another example (a dashboard of TODO lists for different people/tasks where you can add or remove entire TODOs?)